x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/default_config.tsx

```diff
@@ -168,66 +169,71 @@ export const requiredFieldsForActions = [
 ];
 
 export const getSignalsActions = ({
+  apolloClient,
   canUserCRUD,
   hasIndexWrite,
   setEventsLoading,
   setEventsDeleted,
   createTimeline,
   status,
+  updateTimelineIsLoading,
 }: {
+  apolloClient?: ApolloClient<{}>;
   canUserCRUD: boolean;
   hasIndexWrite: boolean;
   setEventsLoading: ({ eventIds, isLoading }: SetEventsLoadingProps) => void;
   setEventsDeleted: ({ eventIds, isDeleted }: SetEventsDeletedProps) => void;
   createTimeline: CreateTimeline;
   status: 'open' | 'closed';
-}): TimelineAction[] => {
-  const actions = [
-    {
-      getAction: ({ eventId, data }: TimelineActionProps): JSX.Element => (
-        <EuiToolTip
-          data-test-subj="send-signal-to-timeline-tool-tip"
-          content={i18n.ACTION_VIEW_IN_TIMELINE}
-        >
-          <EuiButtonIcon
-            data-test-subj={'send-signal-to-timeline-tool-tip'}
-            onClick={() => sendSignalsToTimelineAction({ createTimeline, data: [data] })}
-            iconType="tableDensityNormal"
-            aria-label="Next"
-          />
-        </EuiToolTip>
-      ),
-      id: 'sendSignalToTimeline',
-      width: 26,
-    },
-  ];
-  return canUserCRUD && hasIndexWrite
-    ? [
-        ...actions,
-        {
-          getAction: ({ eventId, data }: TimelineActionProps): JSX.Element => (
-            <EuiToolTip
-              data-test-subj="update-signal-status-tool-tip"
-              content={status === FILTER_OPEN ? i18n.ACTION_OPEN_SIGNAL : i18n.ACTION_CLOSE_SIGNAL}
-            >
-              <EuiButtonIcon
-                data-test-subj={'update-signal-status-button'}
-                onClick={() =>
-                  updateSignalStatusAction({
-                    signalIds: [eventId],
-                    status,
-                    setEventsLoading,
-                    setEventsDeleted,
-                  })
-                }
-                iconType={status === FILTER_OPEN ? 'indexOpen' : 'indexClose'}
-                aria-label="Next"
-              />
-            </EuiToolTip>
-          ),
-          id: 'updateSignalStatus',
-          width: 26,
-        },
-      ]
-    : actions;
-};
+  updateTimelineIsLoading: ({ id, isLoading }: { id: string; isLoading: boolean }) => void;
+}): TimelineAction[] => [
+  {
+    getAction: ({ eventId, ecsData }: TimelineActionProps): JSX.Element => (
+      <EuiToolTip
+        data-test-subj="send-signal-to-timeline-tool-tip"
+        content={i18n.ACTION_VIEW_IN_TIMELINE}
+      >
+        <EuiButtonIcon
+          data-test-subj={'send-signal-to-timeline-tool-tip'}
```

Comment:
This might be legacy but just pointing out that these two HTML elements have the same `data-test-subj`. No changes asked.